### PR TITLE
Dates to iso string

### PR DESCRIPTION
**Pull request by st34k**

### CODE REVIEW

File: packages/base-dao/test/baseTestForDAOs.js [hunk 1]
```diff
@@ -258,12 +258,12 @@ function randomValuesByType(dataType) {
     // Hardcoded values, e.g. enum.
     return [dataType[0], dataType[1]];
   }
-  const timeZoneSuffix = '';
+
   switch (dataType) {
     case 'date':
-      return [`${momentRandom().format('YYYY-MM-DD')}${timeZoneSuffix}`, `${momentRandom().format('YYYY-MM-DD')}${timeZoneSuffix}`];
+      return [`${momentRandom().format('YYYY-MM-DD[T]00:00:00.000Z')}`, `${momentRandom().format('YYYY-MM-DD[T]00:00:00.000Z')}`];
```
Comment (Grishezz) [inline packages/base-dao/test/baseTestForDAOs.js]: what will happen if some entity run a test on a field whose time component matters (that is, date and not dateonly)? e.g. calendar booking...

File: packages/sequelize-model/src/sequelizeInit.js [hunk 2]
```diff
@@ -32,13 +32,14 @@ const sequelize = new Sequelize(config.get('db.name'), config.get('db.user'), co
         runSchemaBasedHooks(entityInput, [jsonToString]);
       },
       afterFind: entityInput => {
-        runSchemaBasedHooks(entityInput, [stringToJson, formatFloat, convertNullToEmptyArray]);
+        runSchemaBasedHooks(entityInput, [stringToJson, formatFloat, convertNullToEmptyArray, formatDate]);
+        console.info('BLAH', entityInput);
```
Comment (Grishezz) [inline packages/sequelize-model/src/sequelizeInit.js]: blah?

File: packages/sequelize-model/src/sequelizeInit.js [hunk 3]
```diff
@@ -73,10 +74,19 @@ Object.keys(sq).forEach(modelName => {
 sq.sequelize = sequelize;
 sq.Sequelize = Sequelize;
 
+function handleRelatedEntityArray(fieldValues, hooks) {
```
Comment (Grishezz) [inline packages/sequelize-model/src/sequelizeInit.js]: please move this function below runSchemaBasedHooks for better readbility

File: packages/base-dao/test/baseTestForDAOs.js (hunk 1 above)
Comment (st34k) [reply packages/base-dao/test/baseTestForDAOs.js]: fixed to format HH:mm:ss:sssZ

File: packages/sequelize-model/src/sequelizeInit.js (hunk 2 above)
Comment (st34k) [reply packages/sequelize-model/src/sequelizeInit.js]: whoops :)

**Pull request merged**